analysis/analysis.go

```diff
@@ -260,6 +267,21 @@ func parseMetricInterval(logCtx log.Entry, metricDurationString v1alpha1.Duratio
 	return metricInterval, nil
 }
 
+// resolveArgsFromValueFromFieldRef resolves args when using ValueFrom.FieldRef.FieldPath
+// returns argument with default value from lookup
+func resolveArgsFromValueFromFieldRef(arg v1alpha1.Argument, namespace string) v1alpha1.Argument {
+	if arg.ValueFrom.FieldRef != nil && arg.ValueFrom.FieldRef.FieldPath != "" {
+		switch arg.ValueFrom.FieldRef.FieldPath {
+		case "metadata.namespace":
```

Comment:
If I am following this correctly does this mean that only metadata.namespace works?

Reply:
Wondering why this does not seem be called form this path: https://github.com/argoproj/argo-rollouts/blob/c6ba82cca216c7c9713c7cd69dae1ebe25ba33a8/utils/analysis/factory.go#L34